tests/test_main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+from app.main import is_isogram
+
+
+def test_empty_string():
+    assert is_isogram('')
+
+
+def test_mix_upper_with_lower():
```

Comment:
i suppose that it's not necessarily to check this, your test_string_without_repeating includes this check

Reply:
It world be better to rename this test to `test_repeated_sensitive_letters`